Content.Server/GameObjects/Components/Power/ApcNetComponents/PowerReceiverUsers/PoweredLightComponent.cs

```diff
@@ -163,13 +172,13 @@ public void UpdateLight(object? sender, EventArgs? e)
         public void UpdateLight()
         {
             var powerReceiver = Owner.GetComponent<PowerReceiverComponent>();
-            var sprite = Owner.GetComponent<SpriteComponent>();
-            var light = Owner.GetComponent<PointLightComponent>();
+            var appearance = Owner.GetComponentOrNull<AppearanceComponent>();
```

Comment:
i know this isn't your doing but i'd like to see these converted into componentdependencies